docker-compose.yml

```diff
@@ -6,28 +6,24 @@ services:
       context: .
       dockerfile: dev.Dockerfile
     env_file: .env.development
-    environment:
-      RUST_LOG: ${RUST_LOG}
     ports:
       - 7878:7878
     volumes:
-      - ${PWD}/src:/app/src
-      - ${PWD}/target:/app/target
-      - ${PWD}/Cargo.lock:/app/Cargo.lock
-      - ${PWD}/Cargo.toml:/app/Cargo.toml
+      - .:/app
+      - hillow_target:/app/target
     restart: always
     depends_on:
       - database
 
   database:
     image: postgres:13
     env_file: .env.development
-    environment:
-      POSTGRES_USER: ${POSTGRES_USER}
-      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-      POSTGRES_DB: ${POSTGRES_DB}
```

Comment:
Why is this being deleted? How do you pass environment variables to PostgreSQL then?
We need to map these with the ones defined in `.env.development` doesn't we?

Reply:
Those are being taken from this configuration:

env_file: .env.development

Reply:
Okey!